CIScripts/Test/PesterLogger/RemoteLogCollector.Tests.ps1

```diff
@@ -0,0 +1,164 @@
+$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path) -replace '\.Tests\.', '.'
+. "$here\$sut"
+
+Describe "RemoteLogCollector" {
+    It "appends collected logs to correct output file" {
+        $Source1 = New-LogSource -Sessions @($Sess1) -Path $DummyLog1
```

Comment:
Protip: You can pass a `T` where argument expects a `T[]` – ie. just `-Sessions $Sess1` – to avoid ascii-fair.